README.md

```diff
@@ -176,12 +176,17 @@ Instead we need to modify layout A's template, and mark the ``description`` anno
 Running Portia
 --------------
 
-Checkout the repository:
+### Dependencies ###
+You might need to install the following packages
 
-    git clone https://github.com/scrapinghub/portia
+    sudo apt-get install libxml2-dev libxslt-dev python-dev zlib1g-dev libffi-dev libssl-dev
```

Comment:
build-essential was already installed on my system so i didn't add it.

zlib was needed i think by numpy. If you really want to know i will start from scratch and show you the log file. Let me know if this is needed.